### ISSUE TITLE
Dark theme toggle lacks accessible label

### ISSUE THREAD

**Issue by ChrisMurphyDev** [Crevitus/marks-big-life-advice#2]
Both from a UX perspective and regarding accessibility, the toggle switch for the dark theme is lacking a label to indicate it's purpose.

**Comment by Crevitus** [comment]
In addition to this, when a user does not have the preferences set on the page, we should use subtle animation to guide their eye to the toggle.
